README.md

```diff
@@ -2,3 +2,80 @@
 Baja web app 2.0
 
 It is recommended to follow the angular style guide written [here](https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md).
+
+## How to install
+
+### Install git
+
+#### On Windows
```

Comment:
Wallace, are you on a Mac? You should add the instructions. They're probably just a couple homebrew packages.